src/common/http/platform/beast/http.cpp

```diff
@@ -458,6 +458,8 @@ static inline error::Error AddProxyAuthHeader(OutgoingRequest &req, BrokenDownUr
 		return ex_encoded_creds.error();
 	}
 	req.SetHeader("Proxy-Authorization", "Basic " + ex_encoded_creds.value());
+	log::Warning(
+		"Avoid using basic authentification if possible, and make sure if it's used, it's through HTTPS");
```

Comment:
```suggestion
		"Avoid using basic authentication if possible, and make sure if it's used, it's through HTTPS");
```

I am not the grammar expert, but I think "authentification" is not a real word in English.

Reply:
My bad, I don't even know how I did that